bin/pycbc_banksim

```diff
@@ -325,9 +338,29 @@ with ctx:
                          low_frequency_cutoff=options.filter_low_frequency_cutoff)
         stilde /= psd
         signals.append((stilde, s_norm, [], signal_params))
+        sig_m1.append(signal_params.mass1)
+        sig_m2.append(signal_params.mass2)
+    sig_m1 = array(sig_m1)
+    sig_m2 = array(sig_m2)
+    sig_tau0, _ = mass1_mass2_to_tau0_tau3(sig_m1, sig_m2,
+                                           options.filter_low_frequency_cutoff)
+    sig_mchirp, _ = mass1_mass2_to_mchirp_eta(sig_m1, sig_m2)
+
+    logging.info("Calculating Mchirp and Tau0")
+    template_m1 = []
+    template_m2 = []
+    for template_params in template_table:
```

Comment:
If you also like less code: I would do this with a couple lines of list comprehension,
```
template_m1 = array([tp.mass1 for tp in template_table])
template_m2 = array([tp.mass2 for tp in template_table])
```